src/static/js/modules/classes/BreakpointHandler.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
 var $ = require( 'jquery' );
+var getBreakpointState = require( '../util/get-breakpoint-state' );
+var _breakpointsConfig = require( '../../config/breakpoints-config' );
```

Comment:
Slight inconsistency in the use of underscores in L:4 and L:5.

Opened https://github.com/cfpb/cfgov-refresh/issues/919 because we have about 50/50 split in our codebase on how we handle this. Please share any thoughts there!